Website/src/rendering/src/assets/css/components/SessionInformation.css

```diff
@@ -0,0 +1,47 @@
+.information-section.session-information {
+  .image-col {
+    .icon {
+      @apply h-4 w-4 mr-2 inline text-black;
+    }
+
+    .info-text {
+      @apply bg-gray-light p-2 flex align-middle content-center;
+    }
+  }
+
+  .description-col {
+    .speaker-name {
+      @apply text-2xl md:text-3xl font-extrabold text-blue;
```

Comment:
Breakpoint Tailwind classes should go on separate `@apply` lines after the general line. One line per breakpoint from smallest to largest.
To apply in the whole file.